tests/test_ssm.py

```diff
@@ -84,7 +89,34 @@ def test_resolve_obj_arg_profile_override(self, mock_get_parameter_value):
         mock_get_parameter_value.return_value = "parameter_value"
         stack_ssm_resolver.resolve()
         mock_get_parameter_value.assert_called_once_with(
-            "/dev/DbPassword", "new_profile"
+            "/dev/DbPassword", region, "new_profile"
+        )
+
+    @patch(
+        "resolver.ssm.SSM._get_parameter_value"
+    )
+    def test_resolve_obj_arg_region_override(self, mock_get_parameter_value):
+        stack = MagicMock(spec=Stack)
+        stack.profile = "test_profile"
+        stack.region = region
+        stack.dependencies = []
+        stack._connection_manager = MagicMock(spec=ConnectionManager)
+
+        custom_region = 'ca-central-1'
+        assert custom_region != region
+
+        stack_ssm_resolver = SSM(
+            {
+                "name": "/dev/DbPassword",
+                "region": custom_region,
```

Comment:
i would like to see a test where the name and profile are provided but the region is not.

Reply:
ok, sorry I misunderstood what you mean.  I thought when you said `Isn't it already covered by the existing tests`, i thought you were referring to unaltered tests that currently exist in the repo.  Looking at it again I see that you modified an existing test to check what I was asking for.